src/widgets/WebexMeeting/WebexMeeting.jsx

```diff
@@ -61,6 +61,12 @@ WebexMeetingWidget.defaultProps = {
 export default withAdapter(withMeeting(WebexMeetingWidget), (props) => {
   const webex = new Webex({
     credentials: props.accessToken,
+    ...(process.env.NODE_ENV === 'production' 
```

Comment:
Help me think this through, is this step processed at build time? `process` won't be on the final bundle right? Because this doesn't exist on a browser.